packages/docs/.vuepress/components/ods-icon.vue

```diff
@@ -0,0 +1,29 @@
+<!-- Usage: <ods-icon icon="complete" /> -->
+
+<template>
+  <div class="ods-icon" v-html="require(`!html-loader!@okta/odyssey-icons/src/${icon}.svg`)" />
```

Comment:
If you'd like to keep parity between the `class` and the icon name, consider using passing the `class` as a [data prop](https://vuejs.org/v2/guide/single-file-components.html) instead.